tests/test_override.conf

```diff
@@ -0,0 +1 @@
+hide-user-data-from-log no
```

Comment:
We already have this file: https://github.com/valkey-io/valkey/blob/unstable/tests/assets/default.conf, which I think mostly covers the same problem you're describing?

Reply:
Ok nice, I didn't see that